HidalgoJessica/intersección.js

```diff
@@ -0,0 +1,18 @@
+//Escribe una función que acepte como parámetro dos arreglos
+//de enteros y regrese los elementos que tienen en común
+let a = [0,1,2,3,4,6,7,9];
+let b = [0,1,2,3,5,7,8,10];
+
+function intersect(a, b) {
+    var d = {};
+    var results = [];
+    for (var i = 0; i < b.length; i++) {
+        d[b[i]] = true;
+    }
+    for (var j = 0; j < a.length; j++) {
+        if (d[a[j]]) 
+            results.push(a[j]);
+    }
+    return results;
+}
```

Comment:
Creo entender lo que intentas xD.

Creo que el approach mas simple es:
```
Por cada elemento del arreglo 'a', recorrer todos los elementos del arreglo 'b'.
tendrias que ir comparando a[i] === b[j]. Si esta condición se cumple, deberías de agregarlo
a tu arreglo auxiliar que se llama 'result'.
```

**Importante**: para lograr iterar todos los elementos en 'b' por cada elemento en 'a' necesitas un ciclo anidado. O sea que tu segundo ciclo **for** debería estar adentro del primero.